### ISSUE TITLE
windows test failure -- test_basic_workflow

### ISSUE THREAD

**Issue by itziakos** [scalative/haas#172]
The test_basic_workflow fails on my local Windows machine (Windows 10, Python 2.7 64bits). It looks to me that the assertion should be `LessEqual` instead of `Less` but I am not totally sure.

**Environment**

```
>pip --version
pip 18.1 from C:\Users\itziakos\.edm\envs\haas-dev\lib\site-packages\pip (python 2.7)

> pip freeze
appdirs==1.4.3
coverage==4.5.3
distribute-remove==1.0.0
docutils==0.14
enum34==1.1.6
funcsigs==1.0.2
-e git+git@github.com:<username>/haas.git@7af274480fc097ddbdf8b282786f0dcd16447c21#egg=haas
mock==2.0.0
packaging==16.8
pbr==5.1.3
pyparsing==2.2.0
six==1.11.0
statistics==1.0.3.5
stevedore==1.11.0
testfixtures==6.7.0
```

**Traceback**
```
======================================================================
FAIL: test_basic_workflow (haas.tests.test_parallel_runner.TestChildResultHandler)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\itziakos\.edm\envs\haas-dev\lib\site-packages\mock\mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "c:\Users\itziakos\Projects\haas\haas\tests\test_parallel_runner.py", line 77, in test_basic_workflow
    self.assertLess(handler.start_time, stop_time)
AssertionError: 1555679920.874 not less than 1555679920.874

----------------------------------------------------------------------
Ran 217 tests in 3.680s
```

**Comment by sjagoe** [comment]
`assertLessEqual` looks reasonable.

**Comment by itziakos** [comment]
fixed by #175
